src/plugins/identity/README.md

```diff
@@ -10,10 +10,136 @@ GitHub and Discourse (potentially with a different username on each service).
 We would like to combine these two identities together, so that we can
 represent that user's combined cred properly. The Identity plugin enables this.
```

Comment:
So I'm not completely sure on what aspects you'd like to see consistency with #1720.
But here's how I'm looking at the suggestions.

Of course I'll check for any technical inaccuracies I can find.
If it seems more concise while having a good (subjective) flow, I'll take it as an improvement.

I'll prefer to keep little side-notes that could lead to "oh, I didn't know that" insights that aren't obvious. For example I would like to keep:
> Unlike most other plugins, the Identity plugin does not add any new contributions to the graph.

As that could lead to: "Oh, I didn't know the expectation was for most plugins to add contributions to the graph." And could help understand why there have been discussions to move Identity to the core, rather than a plugin.

By contrast, this could be removed:
> This folder contains the Identity plugin.

In the `plugins/identity` folder, containing a README titled `SourceCred Identity Plugin`, I'm sure the intended audience has enough info to connect the dots.

Reply:
Keeping the "unlike other plugins", capitalizing "Cred" and rephrased this part:
```diff
- allowing SourceCred to properly compute the user’s cred.
+ allowing SourceCred to properly compute the Cred for this identity.
```